pkg/virt-handler/vm.go

```diff
@@ -621,9 +621,12 @@ func (c *VirtualMachineController) updateVolumeStatusesFromDomain(vmi *v1.Virtua
 			}
 		}
 	}
-	specVolumeMap := make(map[string]v1.Volume)
+	specVolumeMap := make(map[string]bool)
```

Comment:
nit: I think we can just do
```suggestion
	specVolumeMap := make(map[string]struct{})
```